exporter/awsemfexporter/grouped_metric.go

```diff
@@ -83,6 +103,120 @@ func addToGroupedMetric(pmd *pdata.Metric, groupedMetrics map[interface{}]*Group
 			}
 		}
 	}
+
+	return nil
+}
+
+type kubernetesObj struct{
+	Host string `json:"`
+	Labels internalLabelsObj `json:`
+	Namespace_name string `json:`
+	Pod_id string `json:`
+	Pod_name string `json:`
+	Pod_owners internalPodOwnersObj `json:`
+	Service_name string `json:`
+}
+
+type internalLabelsObj struct{
+	App string `json:`
+	Pod_template_hash string `json:`
+}
+
+type internalPodOwnersObj struct{
+	Owner_kind string `json:`
+	Owner_name string `json:`
+}
+
+
+func addKubernetesWrapper(labels map[string]string) error{
+	//create schema
+	schema := kubernetesObj{}
+	schema.Host = "host_name"
+	schema.Labels =
+	internalLabelsObj{
+		App: "app",
+		Pod_template_hash: "pod_template_hash",
```

Comment:
I guess it should be `pod-template-hash`. Dashes inside not underscore. https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-reference-performance-logs-EKS.html